internal/agent/device/device.go

```diff
@@ -195,6 +209,26 @@ func (a *Agent) sync(ctx context.Context, current, desired *v1alpha1.Device) err
 	return nil
 }
 
+func (a *Agent) syncConsoles(ctx context.Context, wg *sync.WaitGroup) {
```

Comment:
this should live in the consile controller shouldn't it

Reply:
OK